dwave_networkx/drawing/zephyr_layout.py

```diff
@@ -192,87 +190,84 @@ def draw_zephyr(G, **kwargs):
 
 
 def draw_zephyr_embedding(G, *args, **kwargs):
-    """Draws an embedding onto Zephyr graph G.
+    """Draws an embedding onto a Zephyr graph ``G``.
 
     Parameters
     ----------
     G : NetworkX graph
-        A Zephyr graph or a subgraph of a Zephyr graph, as produced by
+        :term:`Zephyr` :term:`graph` or a :term:`subgraph` of a Zephyr graph as produced by
         the :func:`dwave_networkx.zephyr_graph` function.
 
     emb : dict
-        Minor-embedding as a dict of form {node: chain, ...}, where ``node`` are
-        nodes in G and ``chain`` are iterables of qubit labels.
+        Minor-embedding as a dict of the form ``{node: chain, ...}``, where ``node`` is
+        nodes in ``G`` and ``chain`` is iterables of qubit labels.
 
     embedded_graph : NetworkX graph (optional, default None)
-        A graph that contains all keys of ``emb`` as nodes.  If specified,
-        edges of G are considered interactions if and only if (1) they
-        exist between two chains of ``emb`` and (2) the keys of the
-        corresponding chains are connected by an edge in  the given graph.
-        If given, only couplers between chains based on this graph are displayed.
+        Graph that contains all keys of the ``emb`` parameter as nodes. If specified,
+        the edges of ``G`` are considered interactions if and only if (1) they
+        exist between two chains of the emb parameter and (2) the keys of the
```

Comment:
```suggestion
        exist between two chains of the ``emb`` parameter and (2) the keys of the
```

Adding a Suggested Change so you can see how you commit these without needing to edit yourself